tasks/sync-granule/index.js

```diff
@@ -68,6 +68,8 @@ exports.syncGranule = function syncGranule(event) {
     duplicateHandling = collection.duplicateHandling;
   }
 
+  log.info(`duplicateHandling: ${duplicateHandling}`);
```

Comment:
This should either come out or be a `log.debug` with something a bit more descriptive